eBoard.md

```diff
@@ -5,6 +5,8 @@ permalink: /about/eboard
 published: true
 ---
 
+{% assign showname = false %}
```

Comment:
[I suggest passing variables directly into the templates](https://jekyllrb.com/docs/includes/) `{% include note.html content="This is my sample note." %}`